snabble/Android-SDK
# Remove SSL Pinning (APPS-1821)

We don't do SSL Pinning anymore. Anywhere.
Due to this, the SSL Pinning has been removed and SSL Pinning for the payment provider Datatrans has been deactivated.

APPS-1821

### How to test?

* Integrate this branch in any of our projects and see if requests still work.
* Check in a project w/ Datatrans as payment provider if checkouts are proceeded as usual.

### Definition of Done


#### App Tests
- ~~Supported languages have been tested~~
- ~~Light-/Dark-Mode has been tested~~
- ~~Edge-Cases have been tested~~
- ~~Android API Levels have been taken care of _(minSdk?)_~~

#### Testing
